stable/acs-engine-autoscaler/README.md

```diff
@@ -107,11 +107,12 @@ Parameter | Description | Default
 `azuresptenantid`| An Azure service principal tenant id | None. You *must* supply one.
 `kubeconfigprivatekey`| The key passed to the `kubeConfigPrivateKey` parameter in your `azuredeploy.parameters.json` generated with `acs-engine` | None. You *must* supply one.
 `clientprivatekey`| The key passed to the `clientPrivateKey` parameter in your `azuredeploy.parameters.json` generated with `acs-engine` | None. You *must* supply one.
-`acsdeployment`| [OPTIONAL] The name of the deployment used to deploy the kubernetes cluster initially. If not provided, it is defaulted to `azuredeploy` | None.
-`sleeptime`| [OPTIONAL] The number of seconds to sleep between scaling loops. If not provided, a default of 60 seconds is used. | 60
+`acsdeployment`| [OPTIONAL] The name of the deployment used to deploy the kubernetes cluster initially. | `azuredeploy`.
+`sleeptime`| [OPTIONAL] The number of seconds to sleep between scaling loops. | 60
 `ignorepools`| [OPTIONAL] A list of comma seperated pool names the autoscaler should ignore. | None.
-`spareagents`| [OPTIONAL] Number of agents per pool that should always remain up. If not provided, a default of 1 agent is used. | 1
-
+`spareagents`| [OPTIONAL] Number of agents per pool that should always remain up. | 1
+`idle-threshold`| [OPTIONAL] Maximum duration (in seconds) an agent can stay idle before being deleted. | 1800 (30 minutes)
+`over-provision`| [OPTIONAL] Number of extra agents to create when scaling out. | 0
```

Comment:
I think the values should be `idlethreshold` and `overprovision` as you have provided in the yaml file.